travis/test_pylint

```diff
@@ -1,9 +1,47 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-"""
-Script to process pylint run and exit with result.
-"""
+
+'''This script runs `pylint` with plugins and special configuration files.
+Use the following types of pylint configuration files:
+
+ 1) Global
+ 2) Pull request
+ 3) Version
+ 4) Beta
+
+
+1) Global
+
+  Used to check enabled messages on this file in all modules of the project.
+  The result affect your build status.
+  File name: `travis_run_pylint.cfg`
+
+
+2) Pull request
+
+  Used to check enabled messages on this file just in modules
+  changed in a pull request of the project.
+  The result affects your build status.
+  File name: `travis_run_pylint_pr.cfg`
+
+
+2) Version
+
+  Used to disable the previously enabled messages of `global` or `pull request`
+  configuration files based on `VERSION` variable environment.
+  File name: `travis_run_pylint_{VERSION}.cfg`
```

Comment:
Maybe it's clearer if we use `travis_run_pylint_exclude_{VERSION}.cfg`.
Now that we're at it, why having `travis_run`in these config file names?
It would be perfectly good to have:
- `pylint_global.cfg`
- `pylint_pr.cfg`
- `pylint_exclude_{VERSION}.cfg` and
- `pylint_beta.cfg`.